cluster/gce/gci/configure-helper.sh

```diff
@@ -1167,6 +1165,9 @@ For Kubernetes copyright and licensing information, see:
 EOF
 }
 
+function override-kubectl {
+    echo "export PATH={KUBE_HOME}/bin:$PATH" > /etc/profile.d/kube_env.sh
```

Comment:
Agreed, and, it looks like `KUBE_HOME` is missing the `$` as well (we would want it to be resolved here)